spring-boot-parent/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>me.snowdrop</groupId>
+    <artifactId>spring-boot-build</artifactId>
+    <version>2.1.6-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>spring-boot-parent</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <fabric8-maven-plugin.version>4.1.0</fabric8-maven-plugin.version>
```

Comment:
Tangent to the core issue but is this version tested? See discussion in https://github.com/snowdrop/rest-http-example/pull/111

Reply:
Though, I guess, this change would address the need to update all examples so that they are all consistent.

Reply:
For now I wanted to get the opinion on the general structure. All the values are taken from original spring-boot-bom or latest http booster.